android-radar/build.gradle

```diff
@@ -53,7 +30,33 @@ dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 }
 
-group = publishedGroupId
-version = libraryVersion
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/cedexis/JCenter/master/bintrayv1.gradle'
\ No newline at end of file
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                pom {
+                    name = 'Android Radar Library'
```

Comment:
Maybe it's a good opportunity for re-branding: "Citrix ITM Android Radar Library"

Reply:
Thanks @ikalligeros I will ask Kostas if this is ok. If it requires approval from others, I leave it as is.

Reply:
Ok I changed the name. I will handle the code rebranding in the next release.